spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkMicroBatchStream.java

```diff
@@ -205,6 +205,11 @@ private boolean shouldProcess(Snapshot snapshot) {
             "Cannot process delete snapshot : %s. Set read option %s to allow skipping snapshots of type delete",
             snapshot.snapshotId(), SparkReadOptions.STREAMING_SKIP_DELETE_SNAPSHOTS);
         return false;
+      case DataOperations.OVERWRITE:
```

Comment:
I don't think that this should use the same configuration to skip deletes and overwrites. Overwrites are different and I think that we should at a minimum have a different property. I would also prefer to have some additional clarity on how we plan to eventually handle this. We could skip overwrites, but what about use cases where they are probably upserts? What about when they're created by copy-on-write MERGE operations?

Reply:
+1. I would prefer they be two separate configs, but also that we have a plan for the longer term to handle sending out these row deltas.

I'd be ok with getting a PR in to ignore `OVERWRITE`, but this isn't something we should ignore in the longer term (or even really the near-to-medium term) as others have mentioned.

Personally I would consider using a schema similar to the delta.io change capture feed that has a dataframe with the before image / after image (row before and after update) and then the type of operation for each row (insert, delete, update_before, update_after).

Reply:
I have added a separate config to skip overwrites.

Reply:
I will discuss with @SreeramGarlapati and will update on the plan to eventually handle upserts.

Reply:
I connected with @SreeramGarlapati to contribute on this PR.

Reply:
I have added a separate config to skip overwrites. I will discuss with @SreeramGarlapati and will update on the plan to eventually handle upserts.

Reply:
I agree with @kbendick - we will invest in emitting change feed - similar to delta format.